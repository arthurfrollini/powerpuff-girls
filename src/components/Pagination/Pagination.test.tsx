import { render, screen } from "@testing-library/react";
import { Pagination } from ".";

import renderer from "react-test-renderer";

describe("Pagination component", () => {
  it("renders correctly", () => {
    render(<Pagination />);

    expect(screen.getByTestId("pagination")).toBeTruthy();
  });

  it("matches snapshot", () => {
    const tree = renderer.create(<Pagination />);

    expect(tree).toMatchSnapshot();
  });
});
