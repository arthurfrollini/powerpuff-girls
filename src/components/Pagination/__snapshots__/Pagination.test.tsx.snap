// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination component matches snapshot 1`] = `
<div
  className="sc-bdvvtL hfEwdR"
  data-testid="pagination"
>
  <button
    className="sc-gsDKAQ jeWMKV"
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#84B5EB",
      }
    }
    value={0}
  >
    1
  </button>
</div>
`;
